sgqlc/endpoint/websocket.py

```diff
@@ -5,18 +5,22 @@
 
 
 class WebSocketEndpoint(BaseEndpoint):
-    '''
+    """
```

Comment:
avoid changing the style, it adds too much patch noise.

Reply:
Yeah, sorry, I had some automatic linting enable in my editor